packages/harfbuzz.rb

```diff
@@ -3,37 +3,33 @@
 class Harfbuzz < Package
   description 'HarfBuzz is an OpenType text shaping engine.'
   homepage 'https://www.freedesktop.org/wiki/Software/HarfBuzz/'
-  @_ver = '2.8.1'
+  @_ver = '2.8.2'
   version @_ver
   license 'Old-MIT, ISC and icu'
-  compatibility 'all'
-  source_url "https://github.com/harfbuzz/harfbuzz/archive/#{@_ver}.tar.gz"
-  source_sha256 'b3f17394c5bccee456172b2b30ddec0bb87e9c5df38b4559a973d14ccd04509d'
+  compatibility 'x86_64'
```

Comment:
I would hate to think freetype is the only thing keeping this from being compatible with other architectures.

Reply:
At this pace, x86_64 is going to be the only compatible architecture in Chromebrew. :)